verification/api/handler.go

```diff
@@ -148,6 +148,14 @@ func parseNonceRequest(nonceParam, nonceSizeParam string) ([]byte, error) {
 		return nil, errors.New("nonce must be valid base64")
 	}
 
+	nonceLen := len(nonce)
+	if nonceLen < 8 || nonceLen > 64 {
+		return nil, fmt.Errorf(
+			"nonce must be between 8 and 64 bytes long; got %d",
+			nonceLen,
```

Comment:
```suggestion
			nonceLen
```

Reply:
That comma is required, as the closing paren is on a new line.

Reply:
Do we need a newline though?

Reply:
"need" is a relative term, but yes. It breaks up a long line. And this makes it easier to modify the call later on (and results in neater diffs) if, e.g.,  we ant to add an additional parameter, or switch their order in the future.

Reply:
Sure, I am fine with this!